sdk/ml/azure-ai-ml/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Release History
 
+## 1.2.1 (Unreleased)
```

Comment:
Next release version should be 1.3.0?

Reply:
Oh it should, I'll change that then. I thought we were doing minor versions.

Reply:
Looking at our changelog, I can see that we've done tertiary versions before. I'm not sure when we choose to do tertiary versus secondary version updates. I'll consult @harneetvirk  about this.

Reply:
According to Harneet, this should be a 1.3 version, since we aren't doing a patch release.